src/app/shared/components/plotly-wrapper/plotly-wrapper.component.ts

```diff
@@ -0,0 +1,90 @@
+import {
+    AfterViewInit,
+    Component,
+    Input,
+    ElementRef,
+    OnChanges,
+    ViewChild,
+    ViewEncapsulation,
+    OnInit,
+    OnDestroy,
+} from '@angular/core';
+import * as React from 'react';
+import { SynapseClient } from 'synapse-react-client';
+import * as ReactDOM from 'react-dom';
+import { SynapseContextProvider } from 'synapse-react-client/dist/utils/SynapseContext';
+import PlotlyWrapper, { PlotlyWrapperProps } from 'synapse-react-client/dist/containers/PlotlyWrapper';
+
+@Component({
+    selector: 'plotlywrapper',
+    templateUrl: './plotly-wrapper.component.html',
+    styleUrls: ['./plotly-wrapper.component.scss'],
+    encapsulation: ViewEncapsulation.None
+})
+
+export class PlotlyWrapperComponent implements OnInit, OnChanges, AfterViewInit, OnDestroy {
+
+    @ViewChild('plotlywrapper') containerRef: ElementRef;
+    @Input() data: [];
+    @Input() layout: {};
+    @Input() config: {};
+    @Input() containerWidth: number;
+    @Input() useResizeHandler: boolean;
+    @Input() plotStyle: React.CSSProperties;
+    @Input() classNames: string;
+
+    classNameList = 'plotlywrapper ';
+    private hasViewLoaded = false;
+
+    constructor() {
+        // empty
+    }
+
+    public ngOnInit() {
+        if (this.classNames) {
+            this.classNameList = this.classNameList + this.classNames;
+        }
+    }
+
+    public ngOnChanges() {
+        this.renderComponent();
+    }
+
+    public ngAfterViewInit() {
+        this.hasViewLoaded = true;
+        this.renderComponent();
+    }
+
+    public ngOnDestroy() {
+        ReactDOM.unmountComponentAtNode(this.containerRef.nativeElement)
+    }
```

Comment:
Added this recently. This is important if using conditional rendering, like `*ngIf`